### PR TITLE
SOX-201 Branch table jobs

### PR DESCRIPTION
**Pull request by zajca**
Jira: SOX-201
KBC: https://github.com/keboola/connection/pull/4495

Before asking for review make sure that:

## Checklist

- [ ] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/SOX/BranchStorageTest.php [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+            [
+                'incremental' => true,
+            ]
+        );
+        $this->assertTableRowsCount(8, $devTableId, $branchClient);
+        $this->assertTableRowsCount(12, $productionTableId, $privilegedClient);
+
+        // test export
+        $results = $branchClient->exportTableAsync($devTableId);
+        // @todo SOX-20 will save file to branch storage and we can use $branchClient but for now we need to use $privilegedClient
```
Comment (jirkasemmler) [inline tests/Backend/SOX/BranchStorageTest.php:111]: jsem to zalinkoval sem https://keboola.atlassian.net/browse/SOX-208

File: tests/Backend/SOX/BranchStorageTest.php [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+        $this->assertFileRowsCount(8, $results['file']['id'], $privilegedClient);
+
+        $results = $privilegedClient->exportTableAsync($productionTableId);
+        $this->assertFileRowsCount(12, $results['file']['id'], $privilegedClient);
+    }
+
+    public function testTableOperations(): void
```
Comment (jirkasemmler) [inline tests/Backend/SOX/BranchStorageTest.php:118]: nechtel bys to prosim rozdelit podle usecasu? jakoze ano, muze to byt jeden velkej test, ale pak nam pada visibilita.

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
cajk, jen bych prosim rozdelil ty operations

File: tests/Backend/SOX/BranchStorageTest.php (hunk 2 above)
Comment (zajca) [reply tests/Backend/SOX/BranchStorageTest.php]: rozděleno

**Pull request merged**